services/director-v2/src/simcore_service_director_v2/modules/instrumentation/_models.py

```diff
@@ -0,0 +1,102 @@
+from dataclasses import dataclass, field
+from typing import Final
+
+from prometheus_client import CollectorRegistry, Histogram
+from prometheus_client.utils import INF
+
+from ...meta import PROJECT_NAME
+
+_MINUTE: Final[int] = 60
+
+_NAMESPACE_METRICS: Final[str] = PROJECT_NAME.replace("-", "_")
+_TIME_BUCKETS: Final[tuple[float, ...]] = (
+    # all seconds in a minute
+    *(x for x in range(60)),
+    # all minutes in an hour
+    *(x * _MINUTE for x in range(1, 60)),
+    INF,
+)
+
+_SUBSYSTEM_DYNAMIC_SIDECAR: Final[str] = "dynamic_sidecar"
+
+_START_STOP_LABELS: Final[tuple[str, ...]] = (
+    "user_id",
+    "wallet_id",
+    "service_key",
+    "service_version",
+)
+
+_SIZE_TIME_LABLES: Final[tuple[str, ...]] = ("byte_size",)
+
+
```

Comment:
let's re-use here stuff from:
https://github.com/ITISFoundation/osparc-simcore/blob/5b35cfe3cb59703c7a43b83f437e1f2397c4ffbb/services/autoscaling/src/simcore_service_autoscaling/modules/instrumentation

I already pre-created some conventions, that I would like to re-use since I am probably the one that will create the dashboards.


so the metrics_namespace is set as the application name with underscores:
- simcore_service_director_v2

then we have the subsystem
- please reuse MetricsBase class for this (or refactor if needed)

sot the metrics will be like if subsystem is set to "dynamic_services":
- simcore_service_director_v2_dynamic_services